Tests/scripts/validate_files_structure.py

```diff
@@ -563,7 +564,7 @@ def validate_added_files(added_files, integration_set, playbook_set, script_set,
         elif re.match(SCRIPT_YML_REGEX, file_path, re.IGNORECASE) or \
                 re.match(SCRIPT_PY_REGEX, file_path, re.IGNORECASE) or \
                 re.match(SCRIPT_JS_REGEX, file_path, re.IGNORECASE):
-            yml_path, code = get_script_package_data(os.path.dirname(file_path))
+            yml_path, code = get_script_package_data(os.path.dirname(file_path) + '/')
```

Comment:
lets change this to: so it will be windows compatible for mister Van Dijk
```suggestion
            yml_path, code = get_script_package_data(os.path.join(os.path.dirname(file_path) , ''))
```

Reply:
let's do these changes in a separate  branch